Tests/Smoketest4.xctestplan

```diff
@@ -19,7 +19,6 @@
         "AuthenticationTest",
         "BaseTestCase",
         "BookmarkingTests",
-        "BrowsingPDFTests",
```

Comment:
According to this change all the tests are going to be enabled for the Smoketest4, does this match with Test Rail?

If not, add in the Smoketest only those tests that run manually as part of the Smokte test. If there are not manual tests for either the Smoke or the Full Functional, please consider adding them in Test Rail and mark them as automated. 
cc @abodea

Reply:
There are no tests in test rail, i will add tests for each of them and will be included in smoke.

Reply:
Tests have been added in test rail and separated in smoke and functional plans